### PR TITLE
Add requiredOption and exitOverride for Chinese Readme

### PR DESCRIPTION
**Pull request by oGsLP**
Add Chinese translation for Commder V4's Readme.md

### CODE REVIEW

File: Readme_zh-CN.md [hunk 1]
```diff
@@ -238,6 +238,24 @@ $ custom --list x,y,z
 [ 'x', 'y', 'z' ]
 ```
 
+### 必需的选项
+
+你可以使用`.requiredOption`指定一个必需的(强制性的)选项，这样的选项在命令行的命令中必须被给出，除非它拥有一个默认的值。另外，这个方法在格式上和使用`.option`一样：The method is otherwise the same as `.option` in format, 采用标志和说明，以及可选的默认值或自定义处理。
```
Comment (shadowspawn) [inline Readme_zh-CN.md]: I think this english text "The method is otherwise the same as `.option` in format," has been translated and can be deleted?

Comment (oGsLP) [reply Readme_zh-CN.md]: Oh..You're right, I made a mistake

**Comment by shadowspawn** [comment]
I'll edit and merge now. 
Thank you  for your contributions.

**Comment by oGsLP** [comment]
By the way, what about adding a well-marked link to the Readme_zh-CN.md in Readme.md. To be honest, I didn't notice there was a Chinese Readme until I'd read all the English version and found a pr for translation by accident. I think it's necessary to let Chinese programmers find the readme easily or it just exists nominally only. 
The guideline maybe like this:
`Commander [中文文档请移步于此](../Readme_zh-CN.md)`

**Comment by shadowspawn** [comment]
I have wondered how the Chinese README gets found too! I saw another project today with multiple translations mentioned at the top of the README which reminded me. I'll have a look at that, and mention you on PR.

**Comment by shadowspawn** [comment]
v4.0.0 has been released. 

Thank you for your contributions.

**Pull request merged**